eo-runtime/src/main/java/org/eolang/PhDefault.java

```diff
@@ -255,6 +267,15 @@ public final void add(final String name, final Phi attr) {
         this.attrs.put(name, new PhWithRho(attr, this));
     }
 
+    /**
+     * Clean up ThreadLocal NESTING variable to prevent memory leaks.
+     * This method should be called when a thread is about to terminate
+     * or when the ThreadLocal is no longer needed.
+     */
+    static void cleansupNesting() {
```

Comment:
@karishka1222 looks like we introduced this method `cleansupNesting()` in order to be only used in tests. Since its redundant in our live code, let's move the method to the tests